feature-libs/customer-ticketing/components/customer-ticketing-list/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from './customer-ticketing-list.module';
+export * from './customer-ticketing-list.component';
+export * from './customer-ticketing-list/customer-ticketing-list-table/customer-ticketing-list-table.component';
```

Comment:
remove these references as you'll only have 

```
export * from './customer-ticketing-list.module';
export * from './customer-ticketing-list.component';
```

and details, but that's in another PR I believe